python/global_scheduler/test/test.py

```diff
@@ -123,38 +149,44 @@ def get_plasma_manager_id(self):
 
     return db_client_id
 
+  def test_task_default_resources(self):
+    task1 = photon.Task(random_driver_id(), random_function_id(), [random_object_id()], 0, random_task_id(), 0)
+    self.assertEqual(task1.required_resources(), [1.0, 0.0])
+    task2 = photon.Task(random_driver_id(), random_function_id(), [random_object_id()], 0, random_task_id(), 0, [1.0, 2.0])
+    self.assertEqual(task2.required_resources(), [1.0, 2.0])
+
   def test_redis_only_single_task(self):
     """
     Tests global scheduler functionality by interacting with Redis and checking
     task state transitions in Redis only. TODO(atumanov): implement.
     """
     # Check precondition for this test:
-    # There should be three db clients, the global scheduler, the local
-    # scheduler, and the plasma manager.
-    self.assertEqual(len(self.redis_client.keys("{}*".format(DB_CLIENT_PREFIX))), 3)
+    # There should be 2n+1 db clients: the global scheduler + one photon and one plasma per node.
+    self.assertEqual(len(self.redis_client.keys("{}*".format(DB_CLIENT_PREFIX))), 2*NUM_CLUSTER_NODES+1)
     db_client_id = self.get_plasma_manager_id()
     assert(db_client_id != None)
     assert(db_client_id.startswith(b"CL:"))
     db_client_id = db_client_id[len(b"CL:"):] # Remove the CL: prefix.
-
+  
```

Comment:
I leave all white space-related cleanup until the very end and run all the code through the linter that formats it for me.